pkg/util/test/util.go

```diff
@@ -3309,15 +3309,27 @@ func ValidateTelemetry(pxImageList map[string]string, cluster *corev1.StorageClu
 
 	if pxVersion.GreaterThanOrEqual(minimumPxVersionCCMGO) && opVersion.GreaterThanOrEqual(opVer1_10) {
 		if shouldTelemetryBeEnabled(cluster) {
-			return ValidateTelemetryV2Enabled(pxImageList, cluster, timeout, interval)
```

Comment:
Just create an uber method called `ValidateTelemetryV2` with a parameter for enabled or disabled and call the corresponding functions inside it.

It would like this:
```
ValidateTelemetryV1(pxImageList, cluster, timeout, interval, shouldTelemetryBeEnabled(cluster))
and 
ValidateTelemetryV2(pxImageList, cluster, timeout, interval, shouldTelemetryBeEnabled(cluster))
```

Reply:
Done